### DISCUSSION TITLE
Error in Set-PnPListItem with -Batch parameter

### DISCUSSION THREAD

**Discussion by mrcvinicius** [pnp/powershell#376] [Q&A] [answered]
I'm trying to execute the script below:

```
$batch = New-PnPBatch
Set-PnPListItem -List 'BatchTest' -Identity 1 -Values @{"Title" = "Edited";} -UpdateType SystemUpdate -Batch $batch
Invoke-PnPBatch -Batch $batch
```

But I'm receiving the following error:

```
Set-PnPListItem : Property set method not found.
At line:2 char:1
+ Set-PnPListItem -List 'BatchTest' -Identity 1 -Values @{"Title" = "Ed ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-PnPListItem], ArgumentException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Lists.SetListItem
```

Get-PnPException return:
[Get-PnPException output.txt](https://github.com/pnp/powershell/files/6085338/Get-PnPException.output.txt)

If I'm execute this script without bash, everything goes right. 
In this script I'm executing Set-PnPListItem just one time for testing, because inside a loop doesn't work either.

**Anyone has any idea what I'm doing wrong?**

### What is the version of the Cmdlet module you are running?

```
Version Name           ModuleType
------- ----           ----------
1.3.18  PnP.PowerShell   Manifest
```

### Which operating system/environment are you running PnP PowerShell on?

- [x] Windows

**Comment by veronicageek** [comment]
@mrcvinicius - I don't see this issue on my side (PS5 / PS7)

![image](https://user-images.githubusercontent.com/25181757/110017588-47adde80-7d1e-11eb-9822-9f01c54acb91.png)

Is there anything particular in your list? What column type, etc...

**Reply by mrcvinicius** [reply]
Thanks for the reply.

There's nothing special in this list, no approval. It's a simple list with two fields:
Title - Single line
Count - Number

I created this list just for testing the option to batch items. 

![image](https://user-images.githubusercontent.com/7624988/110018968-a8312180-7d06-11eb-9a86-3600fc556ba5.png)

I'm using mainly PS5:

![image](https://user-images.githubusercontent.com/7624988/110019262-05c56e00-7d07-11eb-839d-d85746e29ef1.png)

I tested with PS7 and got the same error:

![image](https://user-images.githubusercontent.com/7624988/110021647-c6e4e780-7d09-11eb-9966-dae840483630.png)

**Comment by mrcvinicius** [comment, marked as answer]
After some testes I noticed that what is causing the error are the column "Count". When I remove this column, the script works.

I recreated this field like single text line, multiple lines, date and boolean but the error continues. I think the internal name of the field has some relation with the error.

I tested in another tenant with another list and get the same error. I used Set-PnPListItem with batch parameter in a list with a field called "Count".

Anyway, I just change the name of the field and worked for me.

Thanks for the replies.
